--- a/homeassistant/components/cover/__init__.py	2020-08-29 23:14:12.269215106 -0700
+++ b/homeassistant/components/cover/__init__.py	2020-09-01 21:55:35.268154531 -0700
@@ -78,6 +78,7 @@
 ATTR_CURRENT_TILT_POSITION = "current_tilt_position"
 ATTR_POSITION = "position"
 ATTR_TILT_POSITION = "tilt_position"
+ATTR_TRANSITION = "transition"
 
 
 @bind_hass
@@ -138,7 +139,10 @@
         {
             vol.Required(ATTR_TILT_POSITION): vol.All(
                 vol.Coerce(int), vol.Range(min=0, max=100)
-            )
+            ),
+            vol.Optional(ATTR_TRANSITION): vol.All(
+                vol.Coerce(float), vol.Range(min=0)
+            ),
         },
         "async_set_cover_tilt_position",
         [SUPPORT_SET_TILT_POSITION],
